.github/workflows/testing.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         pinecone_env:
           - prod
           # - staging
-        node_version: [16.x, 18.x, 20.x]
+        node_version: [18.x, 20.x]
```

Comment:
Don't need to do it right now, but we should probably add testing for node 21, 22, etc.

Reply:
Yeah, I think that's a good idea. For right now I'd like to keep the integration runs somewhat fast, and I think covering v18 & v20 at the moment should be sufficient.